package.json

```diff
@@ -8,11 +8,10 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "babel-node `npm bin`/blue-tape ./test/**/*-test.js",
+    "test": "babel-node node_modules/blue-tape/bin/blue-tape test/**/*-test.js",
```

Comment:
We can call each test file separately. They work!

blue-tape/tape here is used for the sake of `glob`. We could do the same and write small JS file that will `glob` files and include them one by one (I initially did that when blue-tape did not include `cli`)